src/schema/v2/partner/partner.ts

```diff
@@ -752,6 +753,41 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLString,
         resolve: ({ collecting_institution }) => collecting_institution,
       },
+      contactsConnection: {
+        description: "A connection of contacts from a Partner.",
+        type: contactsConnection.connectionType,
+        args: pageable({
+          page: {
+            type: GraphQLInt,
+          },
+          size: {
+            type: GraphQLInt,
+          },
+        }),
+        resolve: async ({ id }, args, { partnerContactsLoader }) => {
+          if (!partnerContactsLoader) return null
+
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          const { body } = await partnerContactsLoader(id, {
+            page,
+            size,
+          })
+
+          const totalCount = body.length
```

Comment:
Was hoping I could grab total count from headers but it didnt see to be coming back from Gravity. (the grav endpoint is using `paginate_and_sort` [here](https://github.com/artsy/gravity/blob/main/app/api/v1/partners_contacts_endpoint.rb#L117). Maybe as the requester I'm not properly authenticated 🤔

Reply:
Mentioned above, need a `total_count` argument to the endpoint.